### PR TITLE
Add loading bar component to Content Listing & Node Form pages.

### PR DESCRIPTION
**Pull request by mattgrill**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/136

## Screenshot / UI changes

loading

## Testing instructions

- Make sure the loading bar shows.
- A bunch of the content page needed to be re-indented, sorry?

### CODE REVIEW

File: src/components/05_pages/Content/Content.js [hunk 1]
```diff
@@ -184,288 +186,308 @@ class Content extends Component {
...
+                  page={offset / limit}
+                  onChangePage={this.pageChangeHandler}
+                  rowsPerPageOptions={[limit]}
+                  labelDisplayedRows={({ page }) => `Page: ${page + 1}`}
+                  nextIconButtonProps={{ 'aria-label': 'Next content page.' }}
+                  backIconButtonProps={{
+                    'aria-label': 'Previous content page.',
+                  }}
+                />
+                {!this.props.contentList.length && (
```
Comment (dawehner) [inline src/components/05_pages/Content/Content.js]: Mh, this condition wouldn't be triggered anymore now. Should we remove it?

Comment (mattgrill) [reply src/components/05_pages/Content/Content.js]: I think I refined this a bit in #5243068

**Comment by lauriii** [comment]
Any thoughts on how we could implement the loading animation using the Material UI styleguide? The documentation page for the loading animations: https://material-ui.com/demos/progress/

File: src/components/05_pages/Content/Content.js (hunk 1 above)
Comment (dawehner) [reply src/components/05_pages/Content/Content.js]: 👍

**Comment by mattgrill** [comment]
@lauriii It's not really too far off from the linear progress bars from MaterialUI. I think it's something we can address in a followup?

**Comment by lauriii** [comment]
@mattgrill sounds good!

**Comment by mattgrill** [comment]
I ran the tests locally and don't get the same (any) failures. 

1__matt_grill_corellia____developer_drupal_hosts_86xreact_drupal_vfancy__zsh_

**Pull request merged**